src/main/java/net/minecraftforge/fmp/capabilities/ICapabilityWrapper.java

```diff
@@ -0,0 +1,19 @@
+package net.minecraftforge.fmp.capabilities;
+
+import java.util.Collection;
+
+import net.minecraftforge.common.capabilities.Capability;
+
+/**
+ * Interface that allows mods to provide an implementation of a capability in the case that there's more than one implementation
+ * for one side of a block (multiparts).
+ * 
+ * @see Capability
+ * @see CapabilityWrapperRegistry
+ */
+public interface ICapabilityWrapper<T>
+{
```

Comment:
Would this be better to expose it to the capability system itself? and use that registry? why is this seperate? Whats the use?